api/src/main/resources/liquibase.xml

```diff
@@ -720,4 +720,19 @@
         <comment>Renaming column column_number to bed_column_number in table bed_location_map because column_number is a reserved word in MySQL 8.0.2 and later</comment>
         <renameColumn columnDataType="int" newColumnName="bed_column_number" oldColumnName="column_number" tableName="bed_location_map" />
     </changeSet>
+
+    <changeSet id="BM_202405301229" author="mohankumar">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">select count(*) from location_tag where name='Admission Location'</sqlCheck>
+        </preConditions>
+        <comment>Adding Admission Location to location tag</comment>
+        <sql>
+            INSERT INTO location_tag (name, description, creator, date_created, uuid) VALUES
+            ('Admission Location',
+            'Locations with this tag will be shown for admission and bed management',
+            1,
+            now(),
+            uuid());
```

Comment:
I'm not sure about other implementations, but at our PIH sites we install this tag with initializer, and have it's uuid fixed as `f5b9737b-14d5-402b-8475-dd558808e172`.  This uuid does not need to be different everywhere, right?  Can we fix it to that value?  @dkayiwa thoughts?

Reply:
It does not have to be different.

Reply:
@mseaton The current implementations of Bahmni have it manually created so the UUIDs are different. Anyways I have put a pre-condition, so it wont affect existing implementations. But for new implementations we can fix the UUID to the value to keep it constant.

Reply:
@mohan-13 what I realize is that - in our existing implementations, we set up this location tag with our initializer configuration.  And in our initializer configuration, it has the uuid I listed previously.  Of course, this won't be the same for other implementations, but it's what we use.  

Once we make this change, whenever we install a new system, those new systems will end up in a situation where there are 2 location tags with the same name configured, or will fail if this is not allowed (worse case) or will succeed but will have a different uuid than our configuration is suggesting and a different uuid from our other systems that have been set up previously with initializer.

So actually I am not whether we want to add this after all.  @dkayiwa  / @angshu FYI, thoughts?

Reply:
Just like i said in my very first comment, it is no longer considered community best practice to insert metadata via liquibase sql.

Reply:
There won't be duplicates in existing implementations as the `preCondition` on the changeset will fail if there is a tag with the same name already. This changeSet will only get executed on fresh installations.

Reply:
But we can take the initialiser route if metadata should not be part of the OMOD

Reply:
That would be awesome!!!

Reply:
Yes, given the above I think it might be best to close this PR and go with an alternative approach (eg. use Initializer, or a liquibase changeset in another downstream module).